projects/m1/014-cell-phone-bill/js/index.js

```diff
@@ -0,0 +1,33 @@
+let minute = prompt("How many minutes of call did you use? ")
+let message = prompt("How many messages did you use?")
+let callcentertax= 0.44
+let phoneplan = 15.00   
+let salestax;
+let tot;
+let extraminute;
+let extramessage;
+
+minute = minute * 60
```

Comment:
sarebbe una conversione in secondi? se si, perché non creare una variabile `seconds`? Se la chiami `minute` stai gettando le basi per un futuro bug praticamente impossibile da trovare, perché nessuno penserebbe mai che una variabile minuti in realtà rappresenti i secondi.